pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -253,7 +254,24 @@ func (exp *flowExporter) initFlowExporter(collectorAddr string, collectorProto s
 }
 
 func (exp *flowExporter) sendFlowRecords() error {
-	addAndSendFlowRecord := func(key flowexporter.ConnectionKey, record flowexporter.FlowRecord) error {
+	updateOrSendFlowRecord := func(key flowexporter.ConnectionKey, record flowexporter.FlowRecord) error {
+		// Send the flow record only if the record is timed out with reference to its
+		// last export time. This is done both for active flows(activeFlowTimeout) and
+		// idle flows(idleFlowTimeout). Flows are considered to be idle if the packet
+		// counts do not change when compared to last exported flow record.
```

Comment:
I would suggest:

> Send a flow record if the conditions for either timeout (activeFlowTimeout or idleFlowTimeout) are met. A flow is considered to be idle if its packet counts haven't changed since the last export.